defaults/main.yml

```diff
@@ -7,6 +7,10 @@ retain_extra_time: 12
 
 # TIaaS Deployment
 tiaas_dir: /opt/tiaas
+tiaas_code_dir: "{{ tiaas_dir }}/code"
+tiaas_config_dir: "{{ tiaas_dir }}/config"
+tiaas_static_dir: "{{ tiaas_dir }}/static"
+tiaas_venv_dir: "{{ tiaas_dir }}/venv"
```

Comment:
Please add a default with `tiaas_galaxy_stylesheet: "{{ galaxy_server }}/static/style/base.css"` so for those of us who use the tiaas role in the same playbook we don't have to do anything.

Reply:
Sorry apparently it's `galaxy_server_dir`, haven't looked at that in too long. my bad.

Reply:
Done. Also corrected example in README.
It won't work if the ansible-galaxy layout is `root-dir`: https://github.com/ic4f/ansible-galaxy/blob/master/vars/layout-root-dir.yml#L4 . We *could* take this into consideration, but I fear that would be relying way, way too much on the implementation details of a different role.

Reply:
Sure, sounds good.